cmd/gaia/cli_test/cli_test.go

```diff
@@ -34,42 +34,39 @@ func TestGaiaCLISend(t *testing.T) {
 	flags := fmt.Sprintf("--node=%v --chain-id=%v", servAddr, chainID)
 
 	// start gaiad server
-	cmd, _, _ := tests.GoExecuteT(t, fmt.Sprintf("gaiad start --rpc.laddr=%v", servAddr))
-	defer cmd.Process.Kill()
+	proc := tests.GoExecuteT(t, fmt.Sprintf("gaiad start --rpc.laddr=%v", servAddr))
+	defer proc.Stop(false)
+	time.Sleep(time.Second * 5) // Wait for RPC server to start.
 
 	fooAddr, _ := executeGetAddrPK(t, "gaiacli keys show foo --output=json")
+	fooCech, err := sdk.Bech32CosmosifyAcc(fooAddr)
+	require.NoError(t, err)
 	barAddr, _ := executeGetAddrPK(t, "gaiacli keys show bar --output=json")
+	barCech, err := sdk.Bech32CosmosifyAcc(barAddr)
+	require.NoError(t, err)
 
-	fooBech, err := sdk.Bech32CosmosifyAcc(fooAddr)
-	if err != nil {
-		t.Error(err)
-	}
-	barBech, err := sdk.Bech32CosmosifyAcc(barAddr)
-	if err != nil {
-		t.Error(err)
-	}
-	fooAcc := executeGetAccount(t, fmt.Sprintf("gaiacli account %v %v", fooBech, flags))
+	fooAcc := executeGetAccount(t, fmt.Sprintf("gaiacli account %v %v", fooCech, flags))
 	assert.Equal(t, int64(50), fooAcc.GetCoins().AmountOf("steak"))
 
-	executeWrite(t, fmt.Sprintf("gaiacli send %v --amount=10steak --to=%v --name=foo", flags, barAddr), pass)
-	time.Sleep(time.Second * 3) // waiting for some blocks to pass
+	executeWrite(t, fmt.Sprintf("gaiacli send %v --amount=10steak --to=%v --name=foo", flags, barCech), pass)
+	time.Sleep(time.Second * 2) // waiting for some blocks to pass
 
-	barAcc := executeGetAccount(t, fmt.Sprintf("gaiacli account %v %v", barBech, flags))
+	barAcc := executeGetAccount(t, fmt.Sprintf("gaiacli account %v %v", barCech, flags))
 	assert.Equal(t, int64(10), barAcc.GetCoins().AmountOf("steak"))
-	fooAcc = executeGetAccount(t, fmt.Sprintf("gaiacli account %v %v", fooBech, flags))
+	fooAcc = executeGetAccount(t, fmt.Sprintf("gaiacli account %v %v", fooCech, flags))
 	assert.Equal(t, int64(40), fooAcc.GetCoins().AmountOf("steak"))
 
 	// test autosequencing
-	executeWrite(t, fmt.Sprintf("gaiacli send %v --amount=10steak --to=%v --name=foo", flags, barAddr), pass)
-	time.Sleep(time.Second * 3) // waiting for some blocks to pass
+	executeWrite(t, fmt.Sprintf("gaiacli send %v --amount=10steak --to=%v --name=foo", flags, barCech), pass)
+	time.Sleep(time.Second * 2) // waiting for some blocks to pass
```

Comment:
Any way we can actually ensure the blocks have been committed (query height perhaps)?

Reply:
yeah we are going to do this eventually - there is some code bucky was working on this to enable this